tedi/components/cards/accordion/accordion-item-header/accordion-item-header.component.html

```
@@ -0,0 +1,95 @@
+@if (headerClickable()) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)